### PR TITLE
Eslintprettier

### PR DESCRIPTION
**Pull request by jonenst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,20 @@
...
 
-If you want to test your library integration with a consumer application my-app you have first 
-to build commons-ui via `npm run build` then change the commons-ui dependency in  my-app's package.json from `@gridsuite/commons-ui:'^x.x.x'` 
+If you want to test your library integration with a consumer application my-app you have first
+to build commons-ui via `npm run build` then change the commons-ui dependency in my-app's package.json from `@gridsuite/commons-ui:'^x.x.x'`
 to `@gridsuite/commons-ui:'file:../path/to/the/commons-ui'` then type `npm install` `npm start`.
 
 #### For integrators
-If you want to deploy a new version of commons-ui in the [NPM package registry](https://www.npmjs.com/package/@gridsuite/commons-ui), 
+
```
Comment (ChamseddineBhd) [inline README.md:15]: empty line !

File: src/utils/UserManagerMock.js [hunk 2]
```diff
@@ -52,37 +53,49 @@ export class UserManagerMock {
     }
 
     getUser() {
-        return Promise.resolve(JSON.parse(sessionStorage.getItem("powsybl-gridsuite-mock-user")));
+        return Promise.resolve(
```
Comment (ChamseddineBhd) [inline src/utils/UserManagerMock.js:56]: why this empty line !

File: src/utils/AuthService.js [hunk 3]
```diff
@@ -35,17 +35,23 @@ function initializeAuthenticationProd(dispatch, isSilentRenew, idpSettings) {
...
+                        localStorage.setItem(
+                            'oidc.' + state,
+                            JSON.stringify(storedState)
+                        );
                         sessionStorage.setItem(hackauthoritykey, authority);
                     }
                 }
             }
-            authority = authority || sessionStorage.getItem(hackauthoritykey) || idpSettings.authority;
+            authority =
```
Comment (ChamseddineBhd) [inline src/utils/AuthService.js:51]: empty line !

File: src/components/TopBar/TopBar.js [hunk 4]
```diff
@@ -194,47 +216,70 @@ const TopBar = ({appName, appColor, onParametersClick, onLogoutClick, onLogoClic
                                     <SettingsIcon fontSize="small" />
                                 </ListItemIcon>
                                 <ListItemText>
-                                    <FormattedMessage id="top-bar/settings" defaultMessage={"Settings"}/>
+                                    <FormattedMessage
```
Comment (ChamseddineBhd) [inline src/components/TopBar/TopBar.js:219]: empty line

File: src/components/TopBar/TopBar.js [hunk 5]
```diff
@@ -119,24 +128,33 @@ const TopBar = ({appName, appColor, onParametersClick, onLogoutClick, onLogoClic
...
+    function requestOrExitFullScreen() {
         setAnchorElGeneralMenu(null);
         fullScreenRef.current.fullScreen();
     }
 
     return (
         <AppBar position="static" color="default" className={classes.appBar}>
-            <FullScreen ref={fullScreenRef} onFullScreenChange={onFullScreenChange} onFullScreenError={(e) => console.debug("full screen error : " + e.message)}>
-            </FullScreen>
+            <FullScreen
```
Comment (ChamseddineBhd) [inline src/components/TopBar/TopBar.js:142]: empty line

File: src/components/TopBar/TopBar.js [hunk 6]
```diff
@@ -81,7 +82,15 @@ const StyledMenuItem = withStyles(theme => ({
     },
 }))(MenuItem);
 
-const TopBar = ({appName, appColor, onParametersClick, onLogoutClick, onLogoClick, user, children}) => {
+const TopBar = ({
```
Comment (ChamseddineBhd) [inline src/components/TopBar/TopBar.js:85]: same

File: src/components/Login/Login.js [hunk 7]
```diff
@@ -60,7 +60,11 @@ const Login = ({onLoginClick, disabled}) => {
                     <LockOutlinedIcon />
                 </Avatar>
                 <Typography component="h1" variant="h5">
-                    <FormattedMessage id="login/login" defaultMessage={"login"}/> ?
+                    <FormattedMessage
+                        id="login/login"
+                        defaultMessage={'login'}
+                    />{' '}
```
Comment (ChamseddineBhd) [inline src/components/Login/Login.js:66]: {' '} ?

File: src/components/AuthenticationRouter/AuthenticationRouter.js [hunk 8]
```diff
@@ -5,33 +5,70 @@
...
+                            handleSilentRenewCallback(userManager.instance)
+                        }
+                    />
                 </Route>
                 <Route exact path="/logout-callback">
                     <Redirect to="/" />
                 </Route>
                 <Route>
-                    {userManager.error === null && (<Login disabled={userManager.instance === null} onLoginClick={() => login(location, userManager.instance)}/>)}
+                    {userManager.error === null && (
```
Comment (ChamseddineBhd) [inline src/components/AuthenticationRouter/AuthenticationRouter.js:61]: empty line

File: src/components/AuthenticationRouter/AuthenticationRouter.js [hunk 9]
```diff
@@ -5,33 +5,70 @@
...
+import {
+    handleSigninCallback,
+    handleSilentRenewCallback,
+    login,
+} from '../../utils/AuthService';
+import SilentRenewCallbackHandler from '../SilentRenewCallbackHandler';
+import Login from '../Login';
 
-const AuthenticationRouter = ({userManager, signInCallbackError, dispatch, history, location}) => {
+const AuthenticationRouter = ({
```
Comment (ChamseddineBhd) [inline src/components/AuthenticationRouter/AuthenticationRouter.js:19]: empty line

File: README.md [hunk 10]
```diff
@@ -3,17 +3,20 @@
 Library for sharing GridSuite apps commons components
 
 #### For developers
-The commons-ui library  have a demo app in which you can call your components to test them. 
+
```
Comment (ChamseddineBhd) [inline README.md:6]: empty line !

**Review by ChamseddineBhd — CHANGES_REQUESTED** [review]

File: README.md (hunk 10 above)
Comment (jonenst) [reply README.md:6]: it's prettier's style

File: src/components/Login/Login.js (hunk 7 above)
Comment (jonenst) [reply src/components/Login/Login.js:66]: good catch

**Review by ChamseddineBhd — APPROVED** [review]

File: src/components/Login/Login.js (hunk 7 above)
Comment (jonenst) [reply src/components/Login/Login.js:66]: actually necessary to maintain the whitespace between the year and the question mark

**Pull request merged**